notebooks/Taxi_Dropoff_Reverse_Geocoding.ipynb

```diff
@@ -0,0 +1,504 @@
+{
```

Comment:
<p>I would suggest each image come with some captioning text. For example, this one I suggest adding a markdown block and describe that the input coordinate is stored as separate columns, named 'lon' and 'lat'.</p><p> </p><p>I understand that it looks nice without text when presenting on GTC while you are talking next to it. But could be confusing for users who reads this without that context.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuspatial/pull/1001/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Added captions, let me know if they're descriptive enough!

Reply:
Looks good, thanks!